### ISSUE TITLE
Don't send group change emails to non DC users

### ISSUE THREAD

**Issue by csb0730** [deltachat/deltachat-core#674]
With every group change an event mail is sent to every receipient in the group.

But, for non DC users these emails are confusing and I got often complaints about them.

One approach could be, that these event emails are only sent to known DC users, not to all group members by default.

**Comment by adbenitez** [comment]
I think it isn't confusing, at least not more confusing than no notifications at all, also if a member is added how would the classic MUA users know??
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by csb0730** [comment]
@adbenitez - The problem is simply that this kind of email doesn't exist in standard email correspondence. Users which are not knowing DC are confused about.

I more often got the question from my friends: "Why do You sent this email to me? What shall I do with?"

**Comment by adbenitez** [comment]
well explain to them they are in a Delta Chat group, **this kind of email doesn't exist in standard email correspondence** but they aren't in an standard correspondence group, if I am using a classic MUA and you remove someone from a group, why I would ask you "what I should do"?? isn't obvious that you removed Bob from the group? the only thing you need to say to them is that they are in a Delta Chat group, but the message basically says "hey! I removed Bob from the group" which is much better than if you remove bob and tell my nothing and I later say "Bob how are you?" and they it turns out I am talking to the wind because bob leave the group or was removed.

**Comment by adbenitez** [comment]
the real trouble is:
you remove a member and they get the message that Bob was removed but if they reply to the message they still are replying to Bob

**Comment by STPKITT** [comment]
Such a kind of E-Mail doesn't exist technically in standard E-Mail, however in practice it does, but they are just manually written E-Mails. We just recently had the case where we discussed a birthday present for someone but then someone of the mail receivers wrote an answer informing everybody that he had removed the birthday person so that he won't read about his present beforehand.
What IMO is very important about those group change messages sent by Delta Chat is that they are written in a clear language so that non-tech people exactly understand what's going on and the message should be in the native language of the group members if possible since I know from a lot of Germans that every E-Mail written in English is automatically suspicious to them since a lot of spam-, fishing-mail etc. comes in English.

**Comment by adbenitez** [comment]
@STPKITT exactly!!! if you remove a member in a group of classic MUA users, isn't like you remove them silently you should inform that you removed someone to the rest of the group, it is just useful that Delta Chat does this automatically for us.

**Comment by VP-** [comment]
> the real trouble is:
> you remove a member and they get the message that Bob was removed but if they reply to the message they still are replying to Bob

How about putting Bob in BCC for the removal message?
Or maybe even send a separate mail to Bob, saying "you were removed" instead of "Bob was removed"?

**Comment by adbenitez** [comment]
+1 for BCC 
-1 for "send a separate mail to Bob"
;)

**Comment by csb0730** [comment]
@STPKITT @adbenitez 
Maybe I used the wrong words but I fully agree what you wrote here https://github.com/deltachat/deltachat-core/issues/674?_pjax=%23js-repo-pjax-container#issuecomment-489080414

Explain it more to classic MUA users. That's it :-)

Maybe I should change issue description? I think this would meet more the intension.

**Comment by STPKITT** [comment]
@csb0730 If you agree with our statements & therefor keep sending group change emails maybe you could change the issue description towards something like "Improve (content of) group change emails for non DC users".

**Comment by csb0730** [comment]
Got next complaints :-(

We should not force users to an approach they not want (and in some way not need).
